mne/viz/ui_events.py

```diff
@@ -122,18 +124,23 @@ class ColormapRange(UIEvent):
     kind : str
         Kind of colormap being updated. The Notes section of the drawing
         routine publishing this event should mention the possible kinds.
+    sensor_type : str
```

Comment:
```suggestion
    ch_type : str
```

Reply:
`ch_type` and `cmap` also need to be added under the `Parameters` section of this docstring.